src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -946,6 +946,17 @@ ProcessCLRExceptionNew(IN     PEXCEPTION_RECORD   pExceptionRecord,
 #ifndef HOST_UNIX
     if (!(pExceptionRecord->ExceptionFlags & EXCEPTION_UNWINDING))
     {
+        // If the exception is a breakpoint, but outside of the runtime or managed code,
+        //  let it go.  It is not ours, so someone else will handle it, or we'll see
+        //  it again as an unhandled exception.
```

Comment:
Is this comment correct? I do not think we know that the breakpoint is outside of the runtime or managed code.

Reply:
Oops, forgotten to remove this comment together with the other stuff.